app/views/content_hub/blocks/_double.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="l-content-hub__module l-content-hub__module--hdr-con-lnk-img">
+  <div class="l-content-hub__module-inner">
+    <%= heading_tag block[:header], class: 'content-hub__module-title heading-medium', level: 1 %>
```

Comment:
Don't think this should be level: 1 - at the moment every header on the page is coming out as an `<h1>`